src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotResolvedJavaMethodImpl.java

```diff
@@ -324,6 +324,16 @@ public boolean hasReservedStackAccess() {
         return (getConstMethodFlags() & config().constMethodFlagsReservedStackAccess) != 0;
     }
 
+    /**
```

Comment:
No need for javadoc here - it's inherited from `HotSpotResolvedJavaMethod`.

Reply:
I mostly kept it there for consistency with surrounding methods

Reply:
ok